src/ripple/app/misc/Transaction.h

```diff
@@ -171,6 +172,21 @@ class Transaction
     beast::Journal  j_;
 };
 
+struct TransactionEnvelope
```

Comment:
`TransactionEnvelope` could be too many things, especially considering this is a transaction. I'd rename this.

Reply:
Got rid of TransactionEnvelope.